ingest/cli_parser.py

```diff
@@ -331,6 +331,63 @@ def create_parser():
         "--gene-file", help="Path to .genes.tsv file"
     )
 
+    parser_ingest_differential_expression = subparsers.add_parser(
+        "ingest_differential_expression",
+        help="Indicates author differential expression analysis processing",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--ingest-differential-expression",
+        required=True,
+        action="store_true",
+        help="Indicates that author differential expression analysis should be invoked",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--study-accession",
+        required=True,
+        help="Single study accession associated with provided DE input files.",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-name", required=True, help="Name of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-type", required=True, help="Type of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-scope", required=True, help="Scope of annotation for DE analysis"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--method", default="wilcoxon", help="method for DE"
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--cluster-name", required=True, help="study owner-specified cluster name"
+    )
+
+    # TODO: after end to end support, remove this argument
+    parser_ingest_differential_expression.add_argument(
+        "--cluster-file",
+        required=True,
+        help="Path to cluster file.",
+    )
+
+    parser_ingest_differential_expression.add_argument(
+        "--annotation-file",
+        required=True,
+        help="Absolute or relative path to cell metadata or cluster file of annotations.",
+    )
```

Comment:
This isn't strictly needed, and would be ideal to remove, but doesn't seem worth doing now.

Reply:
Given that they're marked as `required` and also not referenced anywhere, I think we should remove them.  It will require extra effort to source values from the Rails side to set for these, and given that they're not referenced anywhere it does seem better to remove them now.

Reply:
Done.  This was referenced downstream, but with some effort I removed it.